kafka_consumer/datadog_checks/kafka_consumer/kafka_consumer.py

```diff
@@ -1,26 +1,11 @@
 # (C) Datadog, Inc. 2019-present
 # All rights reserved
 # Licensed under Simplified BSD License (see LICENSE)
-import ssl
 
-from kafka import KafkaAdminClient, KafkaClient
-from kafka.oauth.abstract import AbstractTokenProvider
-from six import string_types
-
-from datadog_checks.base import AgentCheck, ConfigurationError, is_affirmative
-from datadog_checks.base.utils.http import AuthTokenOAuthReader
+from datadog_checks.base import AgentCheck, is_affirmative
 from datadog_checks.kafka_consumer.client.kafka_client_factory import make_client
 
-from .constants import BROKER_REQUESTS_BATCH_SIZE, CONTEXT_UPPER_BOUND, DEFAULT_KAFKA_TIMEOUT
-
-
-class OAuthTokenProvider(AbstractTokenProvider):
```

Comment:
This `OAuthTokenProvider` was already present in `kafka_python_client.py` but we just didn't remove it until we could refactor the tests.